test/run-system-test.sh

```diff
@@ -28,8 +28,10 @@ docker compose --env-file .env -p system-test up --wait -d --build --remove-orph
 ./scripts/insert_test_data.sh
 
 # Install pixl_cli and test dependencies
-pip install -e "${PACKAGE_DIR}/pixl_core" && pip install -e "${PACKAGE_DIR}/cli"
-pip install pydicom
+pip install -e "${PACKAGE_DIR}/pixl_core" && \
+  pip install -e "${PACKAGE_DIR}/cli" && \
+  pip install -e "${PACKAGE_DIR}/testutils"
```

Comment:
I think this still needs to be updated

Reply:
Certainly does! Fixed in commit 7cbaa2c9ef9551f0a2f757e907e9a70d5e822935.